instructors.md

```diff
@@ -74,13 +74,20 @@ as long as learners using Windows do not run into roadblocks such as:
 
 *   Time estimates:
     *   @gvwilson: 3 hours
-
+	
+*	How to use the materials in the shell-novice repository (or, lesson planning)
+	* Use the `filesystem` directory for in-workshop exercises and live coding examples.  
+	The easiest way to get can get these files is by cloning the entire shell-novice directory.  Alternatives include creating a zipped copy of the `filesystem` directory and making that available to learners.  
+	* Website: Can give links to learners so they can follow along and see exercises (particularly if you're following the lesson content without many changes).  Make sure to point to website as a post-workshop reference.  
```

Comment:
I personally don't like having the lessons up while I teach, but I have consistent anecdotal evidence indicating that participants like having them up as a way of being able to catch up and/or figure out where we are in the lesson.  So I have mixed feelings but wanted to be fair to the input of other people.